src/Admin/AdminExtension.php

```diff
@@ -28,21 +27,12 @@ class AdminExtension extends AbstractAdminExtension
     protected $actionManager;
 
     /**
-     * @var TokenStorageInterface|SecurityContextInterface
+     * @var TokenStorageInterface
      */
     protected $securityContext;
```

Comment:
Hmm we cannot change the variable name, right?

@greg0ire any ideas?
Duplicate the member var?

Reply:
> Hmm we cannot change the variable name, right?

👎 We could leave it as It is, because we will reduce visibility with the next major release